catalyst-gateway/bin/src/service/api/cardano/rbac/mod.rs

```diff
@@ -1,5 +1,56 @@
 //! RBAC endpoints.
 
-pub(crate) mod chain_root_get;
-pub(crate) mod registrations_get;
-pub(crate) mod role0_chain_root_get;
+use poem_openapi::{param::Query, payload::Json, OpenApi};
+
+use crate::service::common::{
+    auth::{none_or_api_key::NoneOrApiKey, none_or_rbac::NoneOrRBAC},
+    tags::ApiTags,
+    types::cardano::query::stake_or_voter::StakeOrVoter,
+};
+
+mod rbac_registration;
+mod registrations_get;
+mod unprocessable_content;
+
+/// Cardano RBAC API Endpoints
+pub(crate) struct Api;
+
+#[OpenApi(tag = "ApiTags::Cardano")]
+impl Api {
+    #[oai(
+        path = "/draft/rbac/registrations/:catalyst_id",
+        method = "get",
+        operation_id = "rbacRegistrations"
+    )]
+    /// Get RBAC registrations
+    ///
+    /// This endpoint returns RBAC registrations by provided auth Catalyst Id credentials
+    /// or by the `lookup` argument if provided.
+    async fn rbac_registrations_get(
+        &self,
+        /// Stake address to get the RBACE registration for.
+        Query(lookup): Query<Option<StakeOrVoter>>,
+        /// A flag which enalbes returning all corresponded Cip509 registrations
+        Query(detailed): Query<Option<bool>>,
+        /// No Authorization required, but Token permitted.
+        _auth: NoneOrRBAC,
```

Comment:
```suggestion
        auth: NoneOrRBAC,
```

auth shouldn't be unused because it would be the normal source for the registration information to return.